runtime/doc/map.txt

```diff
@@ -71,6 +71,7 @@ modes.
 :ino[remap] {lhs} {rhs}		|mapmode-i|	*:ino* *:inoremap*
 :ln[oremap] {lhs} {rhs}		|mapmode-l|	*:ln*  *:lnoremap*
 :cno[remap] {lhs} {rhs}		|mapmode-c|	*:cno* *:cnoremap*
+:tno[remap] {lhs} {rhs}   |mapmode-t| *:tno* *:tnoremap*
```

Comment:
BTW, there seems to be an alignment issue. Previous entries have an space and a tab, instead of just spaces.